approaches/transformers.md

```diff
@@ -1,7 +1,41 @@
 ---
 grand_parent: Approaches
 parent: Neural machine translation
-layout: coming_soon
 title: Transformers
 description: A deep learning model architecture for neural machine translation
 ---
+
+**Transformers** are a type of neural network architecture that relies only on the [attention](/concepts/attention.md) mechanism.
```

Comment:
"**Transformers** are a type of neural network architecture that *rely* only on..."